os/net/bluetooth/bt_conn.h

```diff
@@ -406,4 +449,21 @@ int bt_conn_le_start_encryption(FAR struct bt_conn_s *conn, uint64_t rand, uint1
 
 int bt_conn_le_conn_update(FAR struct bt_conn_s *conn, uint16_t min, uint16_t max, uint16_t latency, uint16_t timeout);
 
+int bt_conn_addr_le_cmp(const struct bt_conn_s *conn, const bt_addr_le_t *peer);
+
+struct bt_conn_s *bt_conn_lookup_state_le(const bt_addr_le_t *peer, const enum bt_conn_state_e state);
```

Comment:
Please remove this definition in conn_internal.h.

Reply:
Removed bt_conn_lookup_state_le from conn_internal.h.